src/utils/shared.ts

```diff
@@ -0,0 +1,32 @@
+import ms from "ms";
+import { Label } from "../types";
+
+export function calculateLabelValue(label: string): number {
+  const matches = label.match(/\d+/);
+  const number = matches && matches.length > 0 ? parseInt(matches[0]) || 0 : 0;
+  if (label.toLowerCase().includes("priority")) return number;
```

Comment:
We should probably handle this in a new task but these "priority" and "time" labels are intended to be customizable so these shouldn't be hardcoded strings.

Reply:
I actually wonder why is this needed for the `start | stop` commands? What influence do these tags have except that they have to be present on the issue in order to work? `calculateLabelValue` implies some calculation is done with these tags.

Reply:
Good point. This seems to be imported code from the v1 bot, which is later used to generate the price label. Perhaps this should be removed.

Reply:
The below metadata is directly tied to these functions when using the `/start` command in it's current state, taken from [here](https://github.com/ubiquity/devpool-directory-bounties/issues/29#issuecomment-2176741419)

```
<!-- Ubiquity - Assignment - start - 86f94c4
{
  "duration": 3600,
  "priceLabel": {
    "id": 5905388022,
    "node_id": "LA_kwDOJhaJks8AAAABX_0R9g",
    "url": "https://api.github.com/repos/ubiquity/devpool-directory-bounties/labels/Price:%2050%20USD",
    "name": "Price: 50 USD",
    "color": "1f883d",
    "default": false,
    "description": null
  }
}
-->
```

Reply:
I think that multiplier should be applied elsewhere, like on permit generation. Seems simple enough to aggregate all rewards and slap on the multiplier on the very end. 

Reason goes with multiplier, and is read only by `/query`

However, these are not essential functions because we barely used it.